### PR TITLE
[DDW-1092] Make all notification UI/UX the same

### PR DESCRIPTION
**Pull request by daniloprates**
This PR implements all notifications to be on the top of the app and in front of any other element. It also standardizes the notifications, as they were done differently through the app. 

## Screenshots

Screen Shot 2019-12-13 at 22 41 02
Screen Shot 2019-12-13 at 22 41 43
Screen Shot 2019-12-13 at 22 42 22
Screen Shot 2019-12-13 at 22 42 58

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1576359860025000)
- [ ] Test the following notifications:
* Download logs (preparing and success)
* Copy address
* Copy state directory path

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by a-rukin** [comment]
image
this is not related for this PR, but found here 
link color is broken when saving logs

**Comment by nikolaglumac** [comment]
@daniloprates can you please add this fix to this PR? 🙏

File: source/renderer/app/components/notifications/Notification.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  ...$Exact<NotificationMessageProps>,
+  children?: Node,
+  onClose?: Function,
+  isVisible: boolean,
+  order?: 'auto' | number | 'initial' | 'inherit',
+};
+
+export default class Notification extends Component<Props> {
+  static defaultProps = {
+    order: 9999999999999999,
```
Comment (DeeJayElly) [inline source/renderer/app/components/notifications/Notification.js]: Maybe extract to external const variable and reuse it from there?

Comment (daniloprates) [reply source/renderer/app/components/notifications/Notification.js]: @DeeJayElly  ✅ Replaced `order` by `index` received by the container, so the last notification is always on top.

**Comment by DmitriiGaico** [comment]
The Dots to be removed.
Dots

**Comment by a-rukin** [comment]
image
@daniloprates can you fix this too? or I should approve without it?

**Comment by darko-mijic** [comment]
This PR is missing Japanese screenshots which are always very helpful when writing release notes.

**Comment by nikolaglumac** [comment]
@input-output-hk/daedalus can someone please add this missing screenshot? 🙏

**Comment by tomothespian** [comment]
@nikolaglumac I will add missing screenshots

**Comment by tomothespian** [comment]
@nikolaglumac @darko-mijic PR screenshots updated!

**Pull request merged**